lib/rubocop/cop/chef/deprecation/chocolatey_package_uninstall_action.rb

```diff
@@ -34,24 +34,21 @@ module ChefDeprecations
         #     action :remove
         #   end
         #
-        class ChocolateyPackageUninstallAction < Cop
+        class ChocolateyPackageUninstallAction < Base
           include RuboCop::Chef::CookbookHelpers
 
           MSG = 'Use the :remove action in the chocolatey_package resource instead of :uninstall which was removed in Chef Infra Client 14+'
 
+          extend AutoCorrector
           def on_block(node)
             match_property_in_resource?(:chocolatey_package, 'action', node) do |choco_action|
               choco_action.arguments.each do |action|
-                add_offense(action, location: :expression, message: MSG, severity: :warning) if action.source == ':uninstall'
+                add_offense(action, message: MSG, severity: :warning) do |corrector|
+                  corrector.replace(action, ':remove')
+                end if action.source == ':uninstall'
```

Comment:
It's a lot easier to follow these by removing the ```if``` on the block at line 48 and instead add ```next unless action.source == ':uninstall' ``` before the ```add_offense```

Reply:
These should be addressed.